nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
But then wouldn't you have to add `scale` to `propose_point` in every Sampler subclass? (We don't need to fix the algorithm in this PR, but I thought it might help in getting around this problem.)